### PR TITLE
Import mstest results for each tests

### PR DESCRIPTION
**Pull request by fpellet**
Save in sonar, execution result of each tests and global execution Time

### CODE REVIEW

File: src/main/java/org/sonar/plugins/dotnet/tests/VisualStudioTestResultsFileParser.java [hunk 1]
```diff
@@ -79,19 +95,65 @@ private void handleCountersTag() {
...
+        throw new IllegalArgumentException("Invalid format " + period);
+      }
+    }
+
+    private void addTestData() {
+      String id = xmlParserHelper.getRequiredAttribute("id");
+
+      goToTag("TestMethod");
+
+      String dllLocation = xmlParserHelper.getRequiredAttribute("codeBase");
```
Comment (dbolkensteyn) [inline src/main/java/org/sonar/plugins/dotnet/tests/VisualStudioTestResultsFileParser.java]: I'm not convined that this is robust enough

**Comment by dbolkensteyn** [comment]
I see the functional value of this PR, but I don't like how it is implemented. I'm closing this PR.

**Comment by jabbera** [comment]
Is someone from sonar source / msft going to implement this or are you expecting someone from the community to re-work this. If you are expecting the community can you provide some insight into how you would like it implemented? 

Thanks!

**Comment by dbolkensteyn** [comment]
@jabbera There are ongoing discussions on this topic, but I'm unable to provide any estimate as to when progress will be made on that. I don't expect further work from the community will accelerate getting this feature in the product.

**Comment by dbolkensteyn** [comment]
FYI, this open ticket tracks the functional need: https://jira.sonarsource.com/browse/TBSTFNP-11

**Pull request closed without merging**